### ISSUE TITLE
[New Feature]: Login/sign in page

### ISSUE THREAD

**Issue by gaurijoshi25** [GameSphere-MultiPlayer/Physi-c-Tech#3]
### 🎮 feature Request

feature logic and basic description
1.create login.html
2.add styles.css
3.create login.js
4.create signup.html and signup.js
navigate to the login page (login.html), enter their username and password, and upon successful login, they will be redirected to the game page (game.html). If they don't have an account, they can navigate to the signup page (signup.html) to create one.

### Point down the features

feature points
there should be a sign in/login page for personalised user interface..this way there could be a leaderboard option too and a track of what they have done.

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE]() of this project.

**Comment by ShreyashKathade** [comment]
Can i work on this ?

**Comment by abhivarma174** [comment]
I have prior experience in front end domain. I can do this efficiently with clean code structure. Kindly assign me this issue.

**Comment by Rish-tech1** [comment]
I would like to work on this issue. Pls assign me this!!

**Comment by Sailza** [comment]
Hi please assign me this task under GSSOC'24

**Comment by Anou25** [comment]
Hi, I would like to work on this issue, please can you assign this to me.

**Comment by Durgesh4993** [comment]
@gaurijoshi25 I don't think so login/signup is a good for gamers, it will become work as barrier. If you have any good approach we can discuss on it...

**Comment by Durgesh4993** [comment]
@Sailza @ShreyashKathade @abhivarma174 @Anou25 @revathishine Please raise any other issue, i'll assign you 

If you have any doubt you can directly send message on our discord (https://discord.gg/rZb46cCMmK), we have created seprate chennals for all projects...

Thank you :)
